homeassistant/components/lightwave/__init__.py

```diff
@@ -45,7 +49,7 @@
                             CONF_PROXY_PORT, default=DEFAULT_PROXY_PORT
                         ): cv.port,
                         vol.Optional(
-                            CONF_PROXY_IP, default=DEFAULT_PROXY_IP
+                            CONF_PROXY_IP, default=DEFAULT_NO_PROXY
```

Comment:
I think we should remove the default instead and check for `None` below.

Reply:
Sorry, my knowledge of voluptuous is not that good, so not sure how to do that.
Do I  set `default=None`, or do I remove the default entirely?

Reply:
Remove the parameter entirely, and change how the config option is looked up below. Ie, use `dict.get(key)` instead of `dict[key]`.